docs/components/compact.md

```diff
@@ -215,13 +215,13 @@ Generally, the maximum memory utilization is exactly the same as for Prometheus
 
 You need to multiply this with X where X is `--compact.concurrency` (by default 1).
 
-**NOTE:** Don't check heap memory only. Prometheus and Thanos compaction leverages `mmap` heavily which is outside of `Go` `runtime` stats. Refer to process / OS memory used rather. On Linux/MacOS Go will also use as much as available, so utilization will be always near limit.
+**NOTE:** Don't check heap memory only. Prometheus and Thanos's compaction leverages `mmap` heavily which is outside of `Go` `runtime` stats. Refer to process / OS memory used rather. On Linux/MacOS Go will also use as much as available, so utilization will be always near limit.
```

Comment:
I feel like it's fine even without possesor as it was (or do it for both Prometheus and Thanos, but it still reads a bit weird to me)
```suggestion
**NOTE:** Don't check heap memory only. Prometheus and Thanos compaction leverages `mmap` heavily which is outside of `Go` `runtime` stats. Refer to process / OS memory used rather. On Linux/MacOS Go will also use as much as available, so utilization will be always near limit.
```